src/App/Resources/AppResources.resx

```diff
@@ -1541,6 +1541,12 @@
   <data name="ThemeDefault" xml:space="preserve">
     <value>Default (System)</value>
   </data>
+  <data name="AutoDarkTheme" xml:space="preserve">
+    <value>Default Dark Theme</value>
+  </data>
+  <data name="AutoDarkThemeDescription" xml:space="preserve">
+    <value>Choose the dark theme to use when using Default (System) theme while your device's dark mode is enabled</value>
+  </data>
```

Comment:
Even though, here I think it's ok to have a key like this, if we change the above this should change to `DefaultDarkThemeDescription`